client/webserver/site/.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     'no-alert': 'error',
     'no-eval': 'error',
     'no-implied-eval': 'error',
-    'object-shorthand': 'off'
+    'object-shorthand': 'off',
+    'quote-props': ["error", "consistent"]
```

Comment:
Was the the `'ar': ar` thing? I left the watch script running when I switched to this branch, and it complained that it wanted the shorthand `ar`. This change is fine though.

Reply:
> Was the the `'ar': ar` thing? I left the watch script running when I switched to this branch, and it complained that it wanted the shorthand `ar`. This change is fine though.

The entry in `localesMap` in locales.ts?  The lang tag is just `"ar"` because there's no region for this Arabic translation, just language.

Reply:
> The entry in `localesMap` in locales.ts?

Yeah. Linter wanted
```Typescript
const localesMap: Record<string, Locale> = {
  ...
  'de-de': deDE,
  'ar': ar
}
```
to be
```Typescript
const localesMap: Record<string, Locale> = {
  ...
  'de-de': deDE,
  ar
}
```
for some reason.

Reply:
Ohhhh